stm32-metapac-gen/src/lib.rs

```diff
@@ -72,7 +72,7 @@ impl Gen {
 
             if let Some(bi) = &p.registers {
                 if let Some(old_version) = peripheral_versions.insert(bi.kind.clone(), bi.version.clone()) {
-                    if old_version != bi.version {
+                    if (old_version != bi.version) && (bi.kind != "timer") {
```

Comment:
this won't work when you try to actually use this with embassy-stm32.

If H7 neither matches v1 nor v2, the fix is to make a new version and make all h7 timers use that.

Reply:
Ok, I understand. I will do that then.